sdk/keyvault/Azure.Security.KeyVault.Administration/src/BackupOperation.cs

```diff
@@ -91,6 +93,10 @@ public override Uri Value
             get
             {
 #pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
+                if (!HasCompleted)
+                {
+                    throw new RequestFailedException("The operation is not complete.");
```

Comment:
This should be `InvalidOperationException`, and check the `Status` of your response (if there is) to see if it was cancelled, throwing `OperationCanceledException`:

```csharp
        public override KeyVaultCertificateWithPolicy Value
        {
#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
            get
            {
                if (Properties is null)
                {
                    throw new InvalidOperationException("The operation was deleted so no value is available.");
                }

                if (Properties.Status == "cancelled")
                {
                    throw new OperationCanceledException("The operation was canceled so no value is available.");
                }

                if (Properties.Error != null)
                {
                    throw new InvalidOperationException($"The certificate operation failed: {Properties.Error.Message}");
                }

                return OperationHelpers.GetValue(ref _value);
            }
#pragma warning restore CA1065 // Do not raise exceptions in unexpected locations
        }
```

Reply:
Fixed the exception type - created #13287 to track the addition of the cancellation scenario once we validate that it is a valid status and how it is represented in the model.